lib/traffacct.pm

```diff
@@ -686,10 +684,8 @@ sub traffacct_cgi {
 		if(!$silent) {
 			print("  <table cellspacing='5' cellpadding='0' width='1' bgcolor='$colors->{graph_bg_color}' border='1'>\n");
 			print("  <tr>\n");
-			print("  <td bgcolor='$colors->{title_bg_color}' colspan='1'>\n");
-			print("  <font face='Verdana, sans-serif' color='$colors->{title_fg_color}'>\n");
-			print("    <b>&nbsp;&nbsp;Network traffic<b>\n");
-			print("  </font>\n");
+			print("  <td class='td-title' colspan='1'>\n");
```

Comment:
You removed accidentally the variable `$traffacct->{graphs_per_row}`.

Reply:
It was `colspan='1'` in this line before or am I missing something?

Reply:
Oh, yes. I misread it. Sorry.